static/js/Misc.jsx

```diff
@@ -3288,11 +3288,17 @@ const Autocompleter = ({getSuggestions, showSuggestionsOnSelect, inputPlaceholde
     )
 }
 
+const imgAltTextFactory = (caption) => {
```

Comment:
This isn't a factory method. It's just a method that pulls the proper text from `caption`. Maybe call it `getCaptionAltText`.